### ISSUE TITLE
Readme input mapping 

### ISSUE THREAD

**Issue by cabobalot** [madhephaestus/WiiChuck#11]
The readme input mapping appears to be off by 1.

**Comment by madhephaestus** [comment]
yup, they are all wrong, lol. Would you like to submit a PR correcting the documentation?

**Comment by cabobalot** [comment]
Id be happy to, it will just take an unspecified amount of time for me to
get around to it. I thought id just submit the issue sooner than later.
